recipes/croaring/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+export CPLUS_INCLUDE_PATH=${PREFIX}/include
+export LIBRARY_PATH=${PREFIX}/lib
+
+mkdir -p build
+cd build
+cmake \
+    -DENABLE_ROARING_TESTS=OFF \
+    -DCMAKE_INSTALL_PREFIX=${PREFIX} \
+    -DCMAKE_INSTALL_LIBDIR=${PREFIX}/lib \
+    ${SRC_DIR}
+
+make install
```

Comment:
@fabio-cumbo do you know if this one has a `make test` or similar?

Reply:
yes, it has a `make test` but it fails while compiling tests, this is why I added `-DENABLE_ROARING_TESTS=OFF` in `build.sh`

Reply:
is the failing test unrelatedß

Reply:
problem fixed, I added the `make test` too